### ISSUE TITLE
Calcified gallbladder

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10047]
**Preferred term label:**
Calcified gallbladder

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Porcelain gallbladder

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
This is a synonym of https://github.com/obophenotype/human-phenotype-ontology/issues/10046
